backend/src/main/java/gov/cdc/usds/simplereport/api/model/errors/UnidentifiedSpecimenTypeException.java

```diff
@@ -0,0 +1,31 @@
+package gov.cdc.usds.simplereport.api.model.errors;
+
+import graphql.ErrorClassification;
+import graphql.ErrorType;
+import graphql.GraphQLError;
+import graphql.language.SourceLocation;
+import java.util.Collections;
+import java.util.List;
+
+/** Exception to throw when a specimen type code can't be found */
+public class UnidentifiedSpecimenTypeException extends RuntimeException implements GraphQLError {
+
+  private static final long serialVersionUID = 1L;
+
+  public UnidentifiedSpecimenTypeException(String typeCode) {
+    super(
+        "Specimen with type code "
+            + typeCode
+            + " weren't found. to make sure that type code exists");
```

Comment:
Should this message be `Specimen with type code 01234 wasn't found. Make sure that type code exists`?